easybuild/easyconfigs/t/tqdm/tqdm-4.32.1-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'tqdm'
+version = '4.32.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://github.com/tqdm/tqdm"
+description = """A fast, extensible progress bar for Python and CLI"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a860bf2683fdbb4812fe539a6c22ea3f1777843ea985cb8c3807db448a0f7ab']
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [('Python', '3.7.2')]
```

Comment:
Can this be done with multi_deps for both python 2 and 3?
Same for pyEGA3

Reply:
@akesandgren pyEGA3 works only with python3